src/libstd/sys/unix/ext/net.rs

```diff
@@ -537,8 +456,10 @@ impl SocketAncillary {
 
     /// Add file descriptors to the ancillary data.
     ///
-    /// This operation increases the capacity size.
-    /// Technically, that means this operation adds a control message with the level `SOL_SOCKET` and type `SCM_RIGHTS`.
+    /// The function returns `true` if there was enough space in the buffer.
+    /// If there was not enough space then no file descriptors was appended.
```

Comment:
Since the return value indicates an error condition perhaps the function should be marked `#[must_use]`. Not sure if this is sufficiently idiomatic or a result should be used instead.

Reply:
I am open for both suggestions (Result or `#[must_use]`).

Reply:
There's [some precedent](https://github.com/rust-lang/rust/blob/f8d394e5184fe3af761ea1e5ba73f993cfb36dfe/src/libcore/option.rs#L229) for `#[must_use]` on a bool result, so that should be fine.